### ISSUE TITLE
Fixed line width in JAVA properties filter should be configurable.

### ISSUE THREAD

**Issue by ducharmg** [IBM-Cloud/gp-java-tools#146]
The line width is fixed at 80. This is arbitrary and should be configurable.
Java Resource bundles have no such limitation.

private static final int COLMAX = 80;

**Comment by ducharmg** [comment]
another issue is the use of INDENT
private static final String INDENT = "    ";

it adds extra spaces to the messages.

**Comment by srl295** [comment]
What is the rationale for changing the line width? The files should be machine-parsed and not hand edited.

I think if anything, there could be no line width and no continuation characters used.

**Comment by ducharmg** [comment]
I agree that neither line width or indent characters should be used.
But if we make them configurable, we can remove the continuation but still allow backwards compatibility with existing code.

I would suggest that line width = 0 be interpreted as no line length limit imposed and then the issue of the INDENT characters will be made moot.
